src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -235,6 +235,44 @@
         }
 
         {% if excluded_ajax_paths is defined %}
+
+            if (window.fetch) {
```

Comment:
should we try to detect native fetch here ? If someone uses a fetch polyfill based on XHR (e.g. the polyfill maintained by github), we would register the request twice otherwise (at the fetch level and at the xhr level)